.travis.yml

```diff
@@ -221,15 +221,15 @@ matrix:
         - postgresql
       env: DB=pgsql POSTGRESQL_VERSION=9.4
     - php: hhvm
-      sudo: false
+      sudo: true
       dist: trusty
       addons:
         postgresql: "9.5"
       services:
         - postgresql
       env: DB=pgsql POSTGRESQL_VERSION=9.5
     - php: hhvm
-      sudo: false
+      sudo: true
```

Comment:
This builds is still not able to install dependencies via composer: https://travis-ci.org/doctrine/dbal/jobs/198403520#L467

Is this expected?